dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -980,3 +981,11 @@ CWWKD1098.spec.param.position.err.explanation=When special Jakarta Data defined
  positioned after all the query parameters.
 CWWKD1098.spec.param.position.err.useraction=Move the special Jakarta Data defined \
  parameter types to positions at the end of the repository method signature.
+
+CWWKD1099.first.keyword.incompat=CWWKD1099E: The {0} method of the {1} repository \
+ has a name that includes the First keyword, which is incompatible with the \
+ {2} parameter of the method.
+CWWKD1099.first.keyword.incompat.explanation=The First keyword cannot be used \
```

Comment:
A Query by Method Name method that includes a Limit or PageRequest parameter cannot use the First keyword.

Reply:
@ramkumar-k-9286 Thanks! I added your update under commit https://github.com/OpenLiberty/open-liberty/pull/30413/commits/e3de7abbd3ccf9b3a5053df4166d0ce27145ec62
Please review again and add your approval.